### ISSUE TITLE
Translation bug - AC + BC

### ISSUE THREAD

**Issue by wrimle** [citation-style-language/locales#264]
The terms for BC and AD are reversed in the norwegian langauge files locales-nb-NO.xml and locals-nn-NO.xml. The correct should be:

```
<term name="ad">evt.</term>
<term name="bc">fvt.</term>
```

(evt = etter vår tidsregning = after our time reckoning = after year 0)
(fvt = før vår tidsreging = before our time reckoning = before year 0)

Running pandoc on ubuntu 22.04:
pandoc 2.9.2.1
Compiled with pandoc-types 1.20, texmath 0.12.0.2, skylighting 0.8.5
Default user data directory: /home/rune/.local/share/pandoc or /home/rune/.pandoc
Copyright (C) 2006-2020 John MacFarlane

**Comment by wrimle** [comment]
BTW: BCE and CE should also be fvt and evt respectively:

```
    <term name="bce">fvt.</term>
    <term name="ce">evt.</term>

```

**Comment by denismaier** [comment]
Can you confirm this @dittaeva ?

**Comment by dittaeva** [comment]
evt. and fvt. are religion neutral terms, and it is true that evt. is for years after the common/current era, and fvt. for years before.

However, the way I understand it [ad/bc are Christian terms, and bce/ce are neutral terms](https://www.antidote.info/en/blog/reports/bc-and-ad-bce-and-ce-whats-difference). As we have both in CSL, and both in both English and [Norwegian](https://www.sprakradet.no/svardatabase/sporsmal-og-svar/for-og-etter-var-tidsrekning-fvt.-og-evt/), we should provide both terms in Norwegian too. In my opinion, the correct is:

```
<term name="ad">e.Kr.</term>
<term name="bc">f.Kr.</term>

<term name="bce">fvt.</term>
<term name="ce">evt.</term>
```

**Comment by denismaier** [comment]
Does that sound reasonable @wrimle ? Should we make that change?

**Comment by wrimle** [comment]
What @dittaeva says is true, and I considered suggesting the christian terms for BC and AD. The reason I did not was that the templates seems to use BC when you write a negative number as year (-330 for 330BCE). As I have a preference for using the non-religious terms in this and most cases I chose not to suggest it. But it would not be wrong to do so. The bigger problem at present is that the terms are inverted, so that -300 becomes 300AD.
